list_of_prime_numbers/main.py

```diff
@@ -1,18 +1,33 @@
 
 def _is_prime(number):
```

Comment:
This `_is_prime` function works fine except it also returns "True" for the numbers 1 and below. You could add this:  
```
if number<2:                       
     return False
```